Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16 as builder
+FROM golang:1.18 as builder
```

Comment:
I think this Docker file is not used by any supported component nor any tool we have. I would remove it to avoid confusions like this one.

https://github.com/newrelic/nri-mongodb/pull/62#issuecomment-1168182665

Reply:
Make file should be fine, that DockerFile is running the `make` command which should trigger the building and testing.
The [recommended](https://github.com/newrelic/nri-mongodb#building) way is to use make .

Reply:
Agreed. It's not used anymore. Removed